hwdb.d/60-sensor.hwdb

```diff
@@ -662,6 +662,10 @@ sensor:modalias:acpi:SMO8500*:dmi:bvnAmericanMegatrendsInc.:bvr5.6.5:bd07/25/201
 sensor:modalias:acpi:BOSC0200*:dmi:*svnONE-NETBOOKTECHNOLOGYCO*:pnOne-Mix3Pro:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
+# One-netbook OneMix 3s
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:*:svnDefaultstring:pnDefaultstring:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
```

Comment:
given that this is the same matrix as the previous entry, and the product name is pretty similar too, please merge this and the previous entry. There are plenty other entries in this file if you look for examples how to do that.

Hmm, so it appears the vendor/product name strings are not filled in for this device? What's the full modalias string for the device?

Typically we try to avoid encoding the bios release in the match strings, to not break entries on bios updates. We do this as last resort only, if theres not other way to detect things property.